workspaces/getting-started.md

```diff
@@ -31,9 +31,30 @@ Coder redirects you to an overview page for your workspace during the build
 process. Learn more about the workspace
 [creation parameters](./workspace-params.md).
 
-Your workspace persists in the home directory, updates to new versions of the
-image, and runs custom configuration on startup. Learn about the
-[workspace lifecycle](lifecycle.md).
+![Workspace overview](../assets/workspaces/workspace-overview.png)
+
+Your workspace persists in the home directory, updates itself to new versions of
+the image on which it is built, and runs custom configuration on startup. Learn
+about the [workspace lifecycle](lifecycle.md).
+
+### Workspace status
+
+The workspace overview page displays information regarding the status and
+performance of your workspace.
+
+![Workspace overview](../assets/workspaces/status-chip.png)
+
+The following workspace statuses are available:
+
+- **Running**: Your workspace is running
```

Comment:
I think a diagram showing the different states would be useful here, or in the linked lifecycle page (showing the different actions and possible state transitions)

Reply:
@jawnsy That's a good idea; I'll request that one be made

Reply:
update: working w/ Jeb re: diagrams

Reply:
It's useful even without diagrams for sure! Was just suggesting that we add a state chart to show these states and the possible transitions. I'm not sure the best person to review it though.

I'll share a quick example of what I mean 🙂